aten/src/ATen/native/cuda/BatchLinearAlgebra.cpp

```diff
@@ -1249,6 +1323,124 @@ magma_trans_t to_magma(TransposeType trans) {
   auto storage_##name = pin_memory<type>(size); \
   name = static_cast<type*>(storage_##name.data());
 
+namespace {
+
+template <typename scalar_t>
+void apply_ldl_factor(
```

Comment:
```suggestion
void apply_ldl_factor_looped(
```
to go with `ldl_factor_looped_magma`.

Reply:
I removed "looped" completely, there's no specialized batched variant.
Will add the magma prefix.